controllers/apps/operations/ops_manager.go

```diff
@@ -240,7 +240,7 @@ func (opsMgr *OpsManager) checkAndHandleOpsTimeout(reqCtx intctrlutil.RequestCtx
 	if timeoutSeconds == nil || *timeoutSeconds == 0 {
 		return requeueAfter, nil
 	}
-	timeoutPoint := opsRes.OpsRequest.Status.StartTimestamp.Add(time.Duration(*timeoutSeconds))
+	timeoutPoint := opsRes.OpsRequest.Status.StartTimestamp.Add(time.Duration(*timeoutSeconds) * time.Second)
```

Comment:
Great, time.Duration in Go is nanoseconds, could you please help use to check if there is any other similiar bugs in KubeBlocks's code?